apps/nextjs-website/src/app/profile/personal-data/page.tsx

```diff
@@ -64,20 +65,27 @@ const PersonalData = () => {
   }, [user?.attributes]);
 
   const [editItem, setEditItem] = useState<InfoCardItemProps | null>(null);
+  const [showModal, setShowModal] = useState<'password' | 'email' | null>(null);
```

Comment:
I think "showModal" is too generic as name :thinking:  maybe "showConfirmationModal" fit better this `useState` ?
```suggestion
  const [showConfirmationModal, setShowConfirmationModal] = useState<'password' | 'email' | null>(null);
```
Remember that if you accept this suggestion you need to manually refactor all usages